src/protocol/prss.rs

```diff
@@ -86,6 +88,8 @@ impl IndexedSharedRandomness {
             self.used.insert(index);
         }
 
+        metrics::increment_counter!(INDEXED_PRSS_GENERATED);
```

Comment:
Don't we want a `max()` call here?  Sure, if we are properly sequential, this will work, but I might be interested if the index was surprisingly large.

Reply:
That's a good idea

Reply:
eh, so that would require more work unfortunately - gauges are not yet supported by this infrastructure. It is fairly easy to support and I will add a todo for it